--- multimap.hpp.orig	2026-08-13 19:24:09
+++ multimap.hpp	2026-08-17 17:13:43
@@ -71,7 +71,7 @@
     std::multimap<int8_t, StructExternalSplit> o_esplmultimap;
     for(int j=0; j<100; ++j)
     {
-      auto key = random_value<char>(gen);
+      auto key = random_value<int8_t>(gen);
       o_esplmultimap.insert({key,  { random_value<int>(gen), random_value<int>(gen) }});
       o_esplmultimap.insert({key,  { random_value<int>(gen), random_value<int>(gen) }});
     }
@@ -109,7 +109,7 @@
     { \
       auto const count = InMap.count( pair.first ); \
       CHECK_EQ( count, OutMap.count( pair.first ) ); \
-      auto find = InMap.find( pair.first ); \
+      auto find = InMap.lower_bound( pair.first ); \
       bool found = false; \
       for( size_t i = 0; i < count; ++i, ++find ) \
         found |= find->second == pair.second; \
